pythnet/remote-executor/programs/remote-executor/src/state/governance_payload.rs

```diff
@@ -0,0 +1,90 @@
+use anchor_lang::{prelude::*, solana_program::instruction::Instruction};
+
+use crate::{assert_or_err, error::ExecutorError};
+
+pub const MAGIC_NUMBER : u32 = 0x5054474d;
+#[derive(AnchorDeserialize, AnchorSerialize)]
+pub struct ExecutorPayload{
+    pub header : GovernanceHeader,
+    pub instructions: Vec<InstructionData>,
+
+}
+
+#[derive(AnchorDeserialize, AnchorSerialize, PartialEq)]
+pub enum Module {
+    Executor = 0,
+    Target 
+}
+
+#[derive(AnchorDeserialize, AnchorSerialize, PartialEq)]
+pub enum Action {
+    ExecutePostedVaa = 0,
+}
+
+#[repr(u16)]
+#[derive(AnchorDeserialize, AnchorSerialize, PartialEq)]
+pub enum ReceiverChain {
+    Pythnet = 26,
+}
+
+
+#[derive(AnchorDeserialize, AnchorSerialize)]
+pub struct GovernanceHeader {
+    pub magic_number : u32,
+    pub module : Module,
+    pub action : Action,
+    pub chain : ReceiverChain
+}
```

Comment:
i'm a little confused about this bit -- is there a reason we're using the same message header format here as with the target chain stuff? I don't see an issue with reusing the same format, though i also don't see a benefit (?)

Reply:
I think it's good if we wanted to add extra functionality 🤔

Reply:
I don't have any strong opinions. I think at the very least we should keep the magic number and the module enum.